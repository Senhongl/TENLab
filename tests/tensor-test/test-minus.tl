a = [[1, 2, 3], [4, 5, 6]];
b = [[-1, -2, -3], [-4, -5, -6]];
c = a - b;
print(c);