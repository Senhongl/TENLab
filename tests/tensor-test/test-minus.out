  2   4   6
  8  10  12
[ CPUIntType{2,3} ]
